packages/amplication-data-service-generator/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "target": "es2017",
-    "lib": ["es2020"],
+    "lib": ["es2020", "dom"],
```

Comment:
Why did we change this in the PR?

Reply:
@yuval-hazaz I tried to remove it but then when I run npm run build in the root project I am getting this error

![image](https://user-images.githubusercontent.com/39680385/175992964-0f9603a6-4b9e-47d5-b062-bc8db28c5d0e.png)